src/utils/locale.ts

```diff
@@ -62,6 +147,39 @@ export const isRTLLocale = (locale: string): boolean => RTL_LOCALES.includes(loc
 export const getDir = (locale: string): Direction =>
   isRTLLocale(locale) ? Direction.RTL : Direction.LTR;
 
+/**
+ * Get direction of language by language id
+ *
+ * @param {number} languageId
+ * @returns {string}
+ */
+export const getLanguageDirectionById = (languageId: number): string => {
```

Comment:
instead of having `getLanguageFontById` and `getLanguageFontById` together and calling them twice to just access two different keys why don't we combine them in 1 method e.g. `getLanguageDataById` or something similar